job_templates/cyclic_cc_pt/config_fed_client.conf

```diff
@@ -88,12 +86,26 @@
       # This "launcher" component
       id = "launcher"
 
-      # the name of component is SubprocessLauncher and path is the class path
+      # the class path of the component
       path = "nvflare.app_common.launchers.subprocess_launcher.SubprocessLauncher"
 
       # the launcher will invoke the script
       args.script = "python3 custom/{app_script}  {app_config} "
     }
+    {
+      id = "pipe"
+
+      path = "nvflare.fuel.utils.pipe.file_pipe.FilePipe"
+
+      args {
+        mode = "PASSIVE"
```

Comment:
Should we add a comment describing what `mode` means?

Reply:
good suggestion! added!